### PR TITLE
exporter: add namespace to internal metric label

### PR DESCRIPTION
**Pull request by ridwanmsharif**
This change makes the metric label a URI so that OTLP gauges don't accidentally conflict with this label. We will document in the Ops Agent that this is a reserved label key.

### CODE REVIEW

File: exporter/collector/integrationtest/testdata/fixtures/metrics/untyped_gauge_expect.json
Comment (damemi) [inline exporter/collector/integrationtest/testdata/fixtures/metrics/untyped_gauge_expect.json:694]: did you run `make fixtures` to update this file? I got the newline added back and I think that will fix the failing test

**Review by damemi — APPROVED** [review]
just need to fix failing diff

File: exporter/collector/integrationtest/testdata/fixtures/metrics/untyped_gauge_expect.json
Comment (ridwanmsharif) [reply exporter/collector/integrationtest/testdata/fixtures/metrics/untyped_gauge_expect.json]: My editor does this a lot when I open the file again 🤦 . Ran `make fixtures` again and that seems to have fixed this

**Comment by ridwanmsharif** [comment]
The E2E tests have not run because it `Needs /gcbrun from a collaborator`. Is this something you can help with @damemi or do I need to be added as a collaborator?

**Comment by ridwanmsharif** [comment]
@damemi @dashpole can someone with write access merge this?

The only failure seems to be codecov but from looking at it, the affected file doesn't actually have a change in coverage.

**Comment by dashpole** [comment]
Added the [stackdriver-admins](https://github.com/orgs/GoogleCloudPlatform/teams/stackdriver-admins) team as triagers so your team shouldn't need gcbrun anymore.

**Comment by dashpole** [comment]
(you can ignore codecov) it is informing, but doesn't block merging.  I'll leave it to mike to merge

**Pull request merged**
